requirements.txt

```diff
@@ -1,32 +1,113 @@
+async-generator==1.10
+async-timeout==4.0.2
+atomicwrites==1.4.0
+attrs==21.4.0
+axe-selenium-python==2.1.6
 bandit==1.7.1
-safety==1.10.3
-werkzeug==2.0.3
-Flask==2.0.2
-flask-session~=0.3.2
-requests>=2,<3
-msal>=1.7,<2
-markupsafe==2.0.1
-connexion[swagger-ui]>=2.13
-python-dotenv==0.19.2
-pytest==7.0.1
-pytest-mock==3.7.0
-deepdiff==5.7.0
-pre-commit==2.16.0
 black==21.12b0
+Brotli==1.0.9
+cachelib==0.8.0
+certifi==2022.5.18.1
+cffi==1.15.0
+cfgv==3.3.1
+chardet==4.0.0
+charset-normalizer==2.0.12
+chromedriver-py==102.0.5005.61
+click==8.1.3
+clickclick==20.10.2
+colorama==0.4.4
+commonmark==0.9.1
+connexion==2.13.1
+cryptography==37.0.2
+cssmin==0.2.0
+deepdiff==5.7.0
+Deprecated==1.2.13
+distlib==0.3.4
+dnspython==2.2.1
+dparse==0.5.1
+email-validator==1.2.1
+filelock==3.7.1
+Flask==2.0.2
+Flask-Assets==2.0
+Flask-Compress==1.10.1
```

Comment:
Why have all these requirements changed?

Reply:
I just ran 'pip freeze > requirements.txt' in my sourced in (virtual environment) terminal, I think @harryyo had entered the dependencies manually?